.github/workflows/synchronize-with-main.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@main
+        with:
+          token: ${{ secrets.PULUMI_BOT_TOKEN }}
+      - name: Push to `master`
+        run: |
+          git push origin HEAD:main
```

Comment:
Is this the right way around? It looks to me like this action runs on `main` due to the branch filter, so presumably you want to push `main` to `master`, which (I think?) would be `git push origin HEAD:master`, right?

Reply:
yeah doh, I should start looking into drinking coffee :coffee: 

This was a bad copy paste, thanks for catching it!